english/109_jumla.md

```diff
@@ -455,12 +462,13 @@ So a modifier phrase with and without **da** can mean a different thing.
 
 #### Modification with verb phrases
 
-Verbs and verb phrases are turned into modifiers by placing **da** or **di** immediately next to the verb.
+Verbs are turned into modifiers by placing **da** or **di** immediately next to the verb.
+Verb phrases can also be made into modifiers in this way, but any objects must be moved to before the verb.
 
-**arte da penta graf**
-– the art of painting pictures  
-**suke da helpa ale**
-– the joy of helping others
+**shutu da grafi pente**
+– the art of picture-painting (the art of painting pictures)  
+**alo di helpe di suka**
```

Comment:
It should be **su** now. The possessive particles are **su** and **da**.

Reply:
okay.

Reply:
it feels like "rose-colored lips" should be **roza rang di labi**.  is that the same as **labi da roza rang**?

Reply:
At least there won't be a separate possessive particle, so yeah, I guess it's accurate.